scale/job/views.py

```diff
@@ -277,57 +278,75 @@ def create_v6(self, request):
+        # Require docker image value
+        docker_image = rest_util.parse_string(request, 'docker_image', required=True)
 
         # Extract the fields that should be updated as keyword arguments
         extra_fields = {}
-        base_fields = JobType.BASE_FIELDS_V6
+        simple_fields = {'icon_code', 'max_scheduled', 'docker_image'}
+        complex_fields = {'configuration', 'manifest'}
         for key, value in request.data.iteritems():
-            if key not in base_fields and key not in JobType.UNEDITABLE_FIELDS_V6:
+            if key in simple_fields:
                 extra_fields[key] = value
+            elif key not in complex_fields:
+                raise InvalidJobField
 
-        try:
-            with transaction.atomic():
-
-                # Create the job type
-                job_type = JobType.objects.create_job_type_v6(manifest_dict=manifest_dict,
-                                                              trigger_rule_dict=trigger_rule_dict,
-                                                              configuration_dict=configuration_dict,
-                                                              **extra_fields)
-
-        except (InvalidJobField, InvalidTriggerType, InvalidTriggerRule, InvalidTriggerMissingConfiguration,
-                InvalidConnection, InvalidSecretsConfiguration, ValueError) as ex:
-            message = 'Unable to create new job type'
-            logger.exception(message)
-            raise BadParameter('%s: %s' % (message, unicode(ex)))
-        except InvalidSeedManifestDefinition as ex:
-            message = 'Job type interface invalid'
-            logger.exception(message)
-            raise BadParameter('%s: %s' % (message, unicode(ex)))
-        except InvalidJobConfiguration as ex:
-            message = 'Job type configuration invalid'
-            logger.exception(message)
-            raise BadParameter('%s: %s' % (message, unicode(ex)))
-        except InvalidResources as ex:
-            message = 'Job type custom resources invalid'
-            logger.exception(message)
-            raise BadParameter('%s: %s' % (message, unicode(ex)))
+        name = manifest_dict['job']['name']
+        version = manifest_dict['job']['jobVersion']
+        
+        existing_job_type = JobType.objects.filter(name=name, version=version).first()
+        if not existing_job_type:
+            try:
+                with transaction.atomic():
+    
+                    # Create the job type
+                    job_type = JobType.objects.create_job_type_v6(manifest_dict=manifest_dict,
+                                                                  trigger_rule_dict=None,
+                                                                  configuration_dict=configuration_dict,
+                                                                  **extra_fields)
+    
+            except (InvalidJobField, InvalidTriggerType, InvalidTriggerRule, InvalidTriggerMissingConfiguration,
```

Comment:
Remove the `**extra_fields` and remove catching these exceptions, except `InvalidSecretsConfiguration`.